### PR TITLE
Consolidate re-queuing in a single place

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What this PR does / why we need it:

- Consolidate re-queuing in a single place (after finishing the scheduling cycle).
- Fix error messages

#### Which issue(s) this PR fixes:

Follow up to #32

### CODE REVIEW

File: pkg/scheduler/scheduler.go [hunk 1]
```diff
@@ -111,7 +117,7 @@ func (s *Scheduler) schedule(ctx context.Context) {
 			continue
 		}
 		if s.queues.RequeueWorkload(ctx, &w) {
-			log.V(2).Info("Workload requeued", "workload", klog.KObj(w.Obj), "queue", klog.KRef(w.Obj.Namespace, w.Obj.Spec.QueueName))
+			log.V(2).Info("Workload requeued", "queuedWorkload", klog.KObj(w.Obj), "queue", klog.KRef(w.Obj.Namespace, w.Obj.Spec.QueueName))
```
Comment (denkensk) [inline pkg/scheduler/scheduler.go:120]: is log inited with `queuedWorkload` in L101?

Comment (alculquicondor) [reply pkg/scheduler/scheduler.go:120]: No, because that log is overwritten inside the loop (`:=`)

Comment (denkensk) [reply pkg/scheduler/scheduler.go:120]: Sorry, I'm missing some code.

**Comment by denkensk** [comment]
LGTM

**Review by ArangoGutierrez — COMMENTED** [review]
/lgtm

**Pull request merged**